eventing/README.md

```diff
@@ -1,25 +1,28 @@
 # Knative Eventing
 
-Knative Eventing is a system which is designed to address a common need for
-cloud native development:
-
-1. Services are loosely coupled during development and deployed independently on
-   a variety of platforms (Kubernetes, VMs, SaaS or FaaS).
-1. A producer can generate events before a consumer is listening, and a consumer
-   can express an interest in an event or class of events that is not yet being
-   produced.
-1. Services can be connected to create new applications
-   - without modifying producer or consumer.
-   - with the ability to select a specific subset of events from a particular
-     producer
-
-The above concerns are consistent with the
-[design goals of CloudEvents](https://github.com/cloudevents/spec/blob/master/spec.md#design-goals),
-a common specification for cross-service interoperability being developed by the
-CNCF Serverless WG.
+Knative Eventing is a system that is designed to address a common need for cloud native development and 
+provides composable primitives to enable late-binding event sources and event consumers.
+
+## Design overview
+
+Knative Eventing is designed around the following goals:
+
+1. Knative Eventing services are loosely coupled. These services can be developed and deployed independently on,
+   and across a variety of platforms (for example Kubernetes, VMs, SaaS or FaaS).
+1. Event producers and event sources are independent. Any producer (or source), can generate events 
+   before there are active event consumers that are listening. Any event consumer can express interest in an 
+   event or class of events, before there are producers that are creating those events.
+1. Other services can be connected to the Eventing system. These services can perform the following functions:
+   - Create new applications without modifying the event producer or event consumer. 
+   - Select and target specific subsets of the events from thier producers.
+1. Ensure cross-service interoperability. Knative Eventing is consistent with the 
+   [CloudEvents](https://github.com/cloudevents/spec/blob/master/spec.md#design-goals)
+   specification that is developed by the [CNCF Serverless WG](https://lists.cncf.io/g/cncf-wg-serverless).
```

Comment:
This is fine, but I'll later (subsequent PR) run this through prettier.io with `prosewrap=always`.